scripts/india_nhm/nhm_maternal_health/README.md

```diff
@@ -0,0 +1,39 @@
+# Maternal Health Data - 2008 to 2020
+        
+## About the dataset
+Maternal Health Data for the years 2008 to 2020. Available in National Health Mission data portal.
+
+Source: Health Information Management System (HIMS), Ministry of Health and Family Welfare
+
+### Download URL
+Available for download as xls and zip files.
```

Comment:
Can we zip up the binary xls files in data directory into a single file?

Reply:
The README file has link to the data portal which has xls and zip files. Do we need to have a zip file here too?